doc/sphinx/proofs/automatic-tactics/logic.rst

```diff
@@ -253,9 +244,13 @@ some incompatibilities.
       The goal is not of the form :g:`t = u`. Especially note that :tacn:`btauto`
       doesn't introduce variables into the context on its own.
 
-.. tacv:: field
-          field_simplify {* @term}
-          field_simplify_eq
+.. tacv:: field {? [ {+ @one_term } ] }
+          field_simplify {? [ {+ @one_term } ] } {+ @one_term } {? in @ident }
+          field_simplify_eq {? [ {+ @one_term } ] } {? in @ident }
+
+   .. todo PR these are already documented in the ring chapter.  I guess these
+      rely on the same proof search mechanism.  Remove entirely, put in a sentence
+      pointing to the ring chapter--what do you suggest?
```

Comment:
Remove the tactics and their description from here. I just checked that there was no original content in here. The only thing I'd like to keep (move to the other chapter) is the example. This example is interesting because it generates a sub-goal to prove that some stuff is nonzero. So far, the two examples provided in the ring chapter (for the `field` tactic) did not have this, so it would be a good complement.

Reply:
Not yet resolved.

Reply:
I remember doing this already, not sure how it was lost.